site/src/render.tsx

```diff
@@ -81,9 +80,12 @@ const skuRender: Render<RenderContext> = {
   }),
 
   renderDocument: ({ headTags, bodyTags, app: { html, helmetContext } }) => {
-    const webFontLinkTags = uniq(
-      flatten(values(themes).map((theme) => theme.webFonts)).map(
-        (font) => font.linkTag,
+    const webFontLinkTags = Array.from(
+      new Set(
+        Object.values(themes)
+          .map((theme) => theme.webFonts)
+          .flat()
```

Comment:
I think this can become a `flatMap`.
```suggestion
          .flatMap((theme) => theme.webFonts)
```